### DISCUSSION TITLE
What happening with the versions?

### DISCUSSION THREAD

**Discussion by Nubro01** [Tribler/tribler#5654] [General] [answered]
On september 23th 202 I found on https://jenkins-ci.tribler.org/job/Build-Tribler_release/job/Build-Win64/ version 7.5.3 and that worked as exprected. Since october version 7.5.1 is mentioned and on the 16 lot's of new version appeared, what is going on, looks a bit like a big mess at your side, very unprofessional, esspecially if these version do not work and abend. Strange thing is that the standard version is 7.5.2, so? Please explain what is going on. on the Tribler normal forum.

**Comment by ichorid** [comment, marked as answer]
Thanks for reporting!
Currently, we're going through a cleanup of our build system. Remind that Jenkins builds are never intended for **any** use beside testing the build system itself.

**Comment by synctext** [comment]
We're now steaming ahead with 10 full-time people in the team. So we're doing major re-workings and fast progress. With monthly releases we're going to keep things stable, but still aim to make sustainable progress. So 24 releases in coming 24 months...
